tools/SourceKit/lib/SwiftLang/SwiftEditorInterfaceGen.cpp

```diff
@@ -381,12 +381,16 @@ SwiftInterfaceGenContext::create(StringRef DocumentName,
   ASTContext &Ctx = CI.getASTContext();
   CloseClangModuleFiles scopedCloseFiles(*Ctx.getClangModuleLoader());
 
-  // Load standard library so that Clang importer can use it.
+  // Load standard library and implict imports so that Clang importer can use
+  // it.
   auto *Stdlib = Ctx.getModuleByIdentifier(Ctx.StdlibModuleName);
   if (!Stdlib) {
     ErrMsg = "Could not load the stdlib module";
     return nullptr;
   }
+  for (auto unloadedImport : CI.getMainModule()->getImplicitImports().unloadedImports) {
+    (void)Ctx.getModule(unloadedImport.module.getModulePath());
+  }
```

Comment:
This is needed so Clang importer can see `MainActor` type which is declared in `_Concurrency` module

Reply:
Seems reasonable, though I'd be happier if someone more knowledgeable about import resolution checked this.

Reply:
Looks reasonable to me

Reply:
I guess we got away with this before because we only really cared about what's in the `Swift` module. This generalization looks good to me.